### PR TITLE
:globe_with_meridians: Fix the zh_cn typo

### PR DESCRIPTION
**Pull request by SettingDust**

### CODE REVIEW

File: src/main/resources/assets/configured/lang/zh_cn.json [hunk 1]
```diff
@@ -6,22 +6,22 @@
     "configured.gui.add_value": "添加...",
     "configured.gui.reset": "重置",
     "configured.gui.remove": "删除",
-    "configured.gui.modify": "更改",
+    "configured.gui.modify": "修改",
     "configured.gui.restore": "还原",
     "configured.gui.select": "选择",
-    "configured.gui.change": "修改",
+    "configured.gui.change": "更改",
```
Comment (SettingDust) [inline src/main/resources/assets/configured/lang/zh_cn.json:12]: 这里的交换的原因是 change 的语义更强于 modify
The reason of this is `change` "stronger" than `modify`

File: src/main/resources/assets/configured/lang/zh_cn.json [hunk 2]
```diff
@@ -6,22 +6,22 @@
...
     "configured.gui.no_permission": "您无权执行此操作",
     "configured.gui.not_installed": "服务器上没有安装Configured",
     "configured.gui.edit_world_config": "选择世界来编辑 %s 配置",
-    "configured.gui.info": "此菜单是由Configured动态生成的。单击此处打开此mod的CurseForge页面!",
-    "configured.gui.server_config_info": "基于服务器对于每个世界的配置都是独立的。这意味着对配置所做的任何更改将仅应用于选定的世界。",
-    "configured.gui.restore_message": "您确定要重置所有值吗？如果您正在玩模组包，此配置可能已被作者修改，恢复将破坏模组的预期体验。 §6这个操作是不可逆的！",
-    "configured.gui.unsaved_changes": "有未保存的更改。确定要停止编辑吗？",
+    "configured.gui.info": "此菜单是由Configured动态生成的，单击此处打开此模组的CurseForge页面！",
+    "configured.gui.server_config_info": "基于服务器对于每个世界的配置都是独立的，这意味着对配置所做的任何更改将仅应用于选定的世界。",
+    "configured.gui.restore_message": "您确定要重置所有值吗？如果您正在游玩整合包，此配置可能已被作者修改，恢复将破坏整合包的预期体验。§6这个操作是不可逆的！",
```
Comment (SettingDust) [inline src/main/resources/assets/configured/lang/zh_cn.json:20]: 原文翻译不通顺，简单润色及采用更易理解的名词

File: src/main/resources/assets/configured/lang/zh_cn.json [hunk 3]
```diff
@@ -6,22 +6,22 @@
...
+    "configured.gui.unsaved_changes": "有未保存的更改，确定要停止编辑吗？",
     "configured.gui.title.client_configuration": "客户端配置",
-    "configured.gui.title.common_configuration": "常用配置",
-    "configured.gui.title.server_configuration": "服务器配置",
+    "configured.gui.title.common_configuration": "通用配置",
+    "configured.gui.title.server_configuration": "服务端配置",
```
Comment (SettingDust) [inline src/main/resources/assets/configured/lang/zh_cn.json:24]: Server here means `server-side`, not `server`

File: src/main/resources/assets/configured/lang/zh_cn.json [hunk 4]
```diff
@@ -6,22 +6,22 @@
...
-    "configured.gui.restore_message": "您确定要重置所有值吗？如果您正在玩模组包，此配置可能已被作者修改，恢复将破坏模组的预期体验。 §6这个操作是不可逆的！",
-    "configured.gui.unsaved_changes": "有未保存的更改。确定要停止编辑吗？",
+    "configured.gui.info": "此菜单是由Configured动态生成的，单击此处打开此模组的CurseForge页面！",
+    "configured.gui.server_config_info": "基于服务器对于每个世界的配置都是独立的，这意味着对配置所做的任何更改将仅应用于选定的世界。",
+    "configured.gui.restore_message": "您确定要重置所有值吗？如果您正在游玩整合包，此配置可能已被作者修改，恢复将破坏整合包的预期体验。§6这个操作是不可逆的！",
+    "configured.gui.unsaved_changes": "有未保存的更改，确定要停止编辑吗？",
     "configured.gui.title.client_configuration": "客户端配置",
-    "configured.gui.title.common_configuration": "常用配置",
-    "configured.gui.title.server_configuration": "服务器配置",
+    "configured.gui.title.common_configuration": "通用配置",
```
Comment (SettingDust) [inline src/main/resources/assets/configured/lang/zh_cn.json:23]: `common` here means general, not commonly use

**Review by SettingDust — COMMENTED** [review]
@TexBlock Need review

**Comment by TexBlock** [comment]
@MrCrayfish

**Pull request merged**